port_ocean/cli/cookiecutter/cookiecutter.json

```diff
@@ -6,6 +6,9 @@
     "email": "Your address email <you@example.com>",
     "release_date": "{% now 'local' %}",
     "is_private_integration": true,
+    "port_client_id": "you can find it using: https://docs.getport.io/build-your-software-catalog/custom-integration/api/#find-your-port-credentials",
```

Comment:
I wonder if we can do something like this - https://cli.r-lib.org/reference/links.html

Reply:
i will look into it right now

Reply:
we decided to leave it as is, since terminal does not support markdown